lib/Doctrine/DBAL/Types/DateTimeType.php

```diff
@@ -53,7 +53,7 @@ public function convertToDatabaseValue($value, AbstractPlatform $platform)
             return $value;
         }
 
-        if ($value instanceof \DateTime) {
+        if ($value instanceof \DateTime || $value instanceof \DateTimeImmutable) {
```

Comment:
As far as I know (as experienced) the interface is quite safe to be used here. I've bothered @derickr also on Twitter to help us. Because we all know that it's easier to get people's attention by being annoying HAHAHA 😜